docker/gitconfig

```diff
@@ -0,0 +1,2 @@
+[safe]
```

Comment:
Ever since we made these docker updates, I'm starting to get these errors when developing locally:
```
fatal: detected dubious ownership in repository at '/app'
                                                         To add an exception for this directory, call:

                                                                                                      	git config --global --add safe.directory /app
                                                                                                                                                     ** (MatchError) no match of right hand side value: {"", 128}
    (stdlib 3.14.2.2) erl_eval.erl:453: :erl_eval.expr/5
    (stdlib 3.14.2.2) erl_eval.erl:126: :erl_eval.exprs/5
    (stdlib 3.14.2.2) erl_eval.erl:449: :erl_eval.expr/5
    (elixir 1.13.4) lib/code.ex:404: Code.validated_eval_string/3
    (elixir 1.13.4) lib/config.ex:260: Config.__eval__!/3
    (mix 1.13.4) lib/mix/tasks/loadconfig.ex:54: Mix.Tasks.Loadconfig.load_compile/1
```
this happens when `git log` in the `config/config.exs` file is executed

Reply:
This error was actually one of the motivations to get started working again on the Docker image.
The `fatal: detected dubious ownership in repository at '/app'` was happening here: 
https://user-images.githubusercontent.com/750314/207631812-d0dc9dd8-54d2-4899-a03f-892fa9dbcc90.png
https://github.com/aeternity/ae_mdw/pull/1080

Locally what I am doing and working fine is run the dev image with `docker-compose -f docker-compose-dev.yml up` and then connect to it with a remote shell.